velox/common/memory/tests/MemoryAllocatorTest.cpp

```diff
@@ -741,6 +746,83 @@ TEST_P(MemoryAllocatorTest, externalAdvise) {
   EXPECT_TRUE(instance->checkConsistency());
 }
 
+TEST_P(MemoryAllocatorTest, allocContiguous) {
+  struct {
+    MachinePageCount nonContiguousPages;
```

Comment:
nit: since this test is for contiguous allocations, perhaps, we can drop 'Contiguous' from names.

Reply:
But contiguouAllocation can take non-contiguous allocation as one of the inputs. It is a test parameter so I need to keep "contiguous" in parameter naming.